pinakes/main/auth/views.py

```diff
@@ -42,9 +42,6 @@ def get_object(self):
 class SessionLogoutView(APIView):
     permission_classes = (IsAuthenticated,)
 
-    def get_serializer(self):
```

Comment:
This method was added to address a DRF error reported in console. We will still need a serializer. In stead of return a `None` serializer, we can remove this method, and add
```
from rest_framework.serializers import Serializer
serializer_class = Serializer
```
I have tested and prove it works.

BTW the 500 error occurs only when attempt to GET the logout endpoint which is not allowed.

Reply:
GET endpoint opens DRF Browseable API.

Reply:
So I guess you are referring to the following message in the log:

```
Error #0: SessionLogoutView: unable to guess serializer. This is graceful fallback handling for APIViews. Consider using GenericAPIView as view base class, if view is under your control. Ignoring view for now. 
```

DRF spectacular expects a view that has a serializer, but view classes inherited directly from the `APIView` base class don't have serializers by design. Since serializer is required for generating OpenApi schema only, it can be handled on a DRF spectacular level, by specifying request and response formats in annotation to the view class.

    @extend_schema(
        description="Logout current session",
        tags=["auth"],
        operation_id="logout_create",
        request=None,
        responses={
            status.HTTP_204_NO_CONTENT: OpenApiResponse(
                description="Logout successful"
            )
        },
    )

P.S. Since the view doesn't return any payload, it's probably worth returning `204 No Content` instead of `200 OK`.